www/source/partials/docs/_glossary-builder.html.md.erb

```diff
@@ -1,10 +1,14 @@
 # <a name="glossary-builder" id="glossary-builder" data-magellan-target="glossary-builder">Builder</a>
 
-Builder consists of a continuous build service and a searchable package repository.
+Habitat Builder consists of the Builder service and the Depot.
+
 
 ## Habitat Builder
 
-Users have the option to connect their GitHub repos to Builder to enable continuous builds of their plans as either they checkin new code to GitHub, which will then initiate a new build via a Github hook, or as underlying package dependencies are updated. If  you've added your Habitat plan to the root of your source code repository and a package you depend on (e.g. openssl, ruby, node) needs to be rebuilt due to a new release, your software will be rebuilt automatically and posted to your software's `unstable` channel for you to review and promote per your regular procedure.
+Users have the option to connect their GitHub repositories to Builder to enable continuous builds of their plans. 
+ as either they check-in new code to GitHub, which will then initiate a new build via a Github hook, or as underlying package dependencies are updated. 
+ If you've added your Habitat plan to the root of your source code repository and any of the packages your project depends on (for example, openssl, ruby, or node) has a new release, Builder will automatically rebuild your software and post the updated package to your project's `unstable` channel, where you can review and promote it according to your regular release procedure.
```

Comment:
We should clarify
> has a new release
means a new release of a habitat package (through the work of the `core-plans` maintainers). It's not magic.